packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -403,20 +403,71 @@ backbone = keras_nlp.models.Backbone.from_preset("hf://${model.id}")
 `,
 ];
 
-export const keras_hub = (model: ModelData): string[] => [
-	`# Available backend options are: "jax", "torch", "tensorflow".
+export function keras_hub(model: ModelData): string[] {
+	let class_name =
+		// If the model has a task.json config, then the base Task class is known
+		model.config?.keras_hub_task_json?.class_name ??
+		// If only a config.json is present, the base class will be a "backbone"
+		model.config?.keras_hub_config_json?.class_name;
```

Comment:
Related to my comment above, if we can get rid of some logic by parsing only `task.json`, that would be for the best.